libs/header/sync/sync.go

```diff
@@ -276,25 +276,25 @@ func (s *Syncer[H]) findHeaders(ctx context.Context, from, to uint64) ([]H, erro
 	if amount > s.Params.MaxRequestSize {
 		to, amount = from+s.Params.MaxRequestSize, s.Params.MaxRequestSize
 	}
-
```

Comment:
Pls, don't. This empty line separates var definition from the actual logic and enhances readability

Reply:
Pls